File: client/TuttiFrutti/TuttiFruttiCore/src/main/java/com/example/TuttiFruttiCore/Player.java
package com.example.TuttiFruttiCore;


import java.io.Serializable;

public class Player implements Serializable{
    private String registrationId;
    private String name;
    private String fbId;
    private String email;

    public Player(){}
    public Player(String userId) {
        this.setFbId(userId);
    }

    public String getRegistrationId() {
        return registrationId;
    }

    public void setRegistrationId(String registrationId) {
        this.registrationId = registrationId;
    }

    public String getName() {
       if(name == null)
        return "NO TIENE NOMBRE, FIJATE";

        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getFbId() {
        return fbId;
    }

    public void setFbId(String fbId) {
        this.fbId = fbId;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @Override
    public boolean equals(Object object)
    {
        boolean sameSame = false;

        if (object != null && object instanceof Player)
        {
            sameSame = this.fbId.equals(((Player) object).fbId);
        }

        return sameSame;
    }

    @Override
    public int hashCode() {
        return Integer.valueOf(this.fbId);
    }
}

File: web/src/tuttifruttiweb/PlayerBundle/Entity/Player.php
<?php

namespace tuttifruttiweb\PlayerBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Player
 *
 * @ORM\Table()
 * @ORM\Entity
 */
class Player
{
    public function set($data) {
        if (!isset($data)) return;
        foreach ($data AS $key => $value) {
            switch ($key) {
                case 'dateInserted':
                    $dateInserted = \DateTime::createFromFormat('Y-m-d\TH:i:s.u\Z', $value);
                    $this->setDateInserted($dateInserted);
                    break;
                default:
                    $this->{$key} = $value;
                    break;
            }
        }
    }
    
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="string", length=255)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $fbId;

    /**
     * @var string
     *
     * @ORM\Column(name="name", type="string", length=255)
     */
    private $name;
    /**
     * @var string
     *
     * @ORM\Column(name="email", type="string", length=255)
     */
    private $email;
    /**
     * @var string
     *
     * @ORM\Column(name="registrationId", type="string", length=255)
     */
    private $registrationId;
    /**
     * @var string
     *
     * @ORM\Column(name="dateInserted", type="string", length=255)
     */
    private $dateInserted;


    /**
     * Get fbId
     *
     * @return string 
     */
    public function getFbId()
    {
        return $this->fbId;
    }

    /**
     * Set name
     *
     * @param string $name
     * @return Player
     */
    public function setName($name)
    {
        $this->name = $name;

        return $this;
    }

    /**
     * Get name
     *
     * @return string 
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Set email
     *
     * @param string $email
     * @return Player
     */
    public function setEmail($email)
    {
        $this->email = $email;

        return $this;
    }

    /**
     * Get email
     *
     * @return string 
     */
    public function getEmail()
    {
        return $this->email;
    }

    /**
     * Set registrationId
     *
     * @param string $registrationId
     * @return Player
     */
    public function setRegistrationId($registrationId)
    {
        $this->registrationId = $registrationId;

        return $this;
    }

    /**
     * Get registrationId
     *
     * @return string 
     */
    public function getRegistrationId()
    {
        return $this->registrationId;
    }

    /**
     * Set dateInserted
     *
     * @param string $dateInserted
     * @return Player
     */
    public function setDateInserted($dateInserted)
    {
        $this->dateInserted = $dateInserted;

        return $this;
    }

    /**
     * Get dateInserted
     *
     * @return string 
     */
    public function getDateInserted()
    {
        return $this->dateInserted;
    }
}
